DB/projects.json

```
@@ -5398,5 +5398,19 @@
         "tech": ["typescript", "reactjs", "firebase", "mui"]
       }
     ]
+  },
+  {
+    "github_username": "Sushilverma002",  
+    "Social_media": {
+      "LinkedIn": "https://www.linkedin.com/in/sushil-verma-1b228423a/"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Sushilverma002/applanding",
+        "title": "App Landing Page",
+        "description": "An App landing page which shows the features of your app on website.",
+        "tech": ["html", "css", "bootstrap","js"]
```

change to `javascript`

```suggestion
        "tech": ["html", "css", "bootstrap","javascript"]
```